### PR TITLE
Fix #13773 - Export page of Settings is not responsive

### LINKED ISSUE #13773: Export page of Setting is not responsive
### Steps to reproduce
1. From top navigation goto "Setting"
2.  Goto "Export"

### Expected behaviour
Page should be responsive

### Actual behaviour
Not responsive
I think that it is because of options with long text in drop down/list of "export method"

screenshot_20171024-234038

**phpMyAdmin version:** Online Demo from git master

**Browser:** Latest Stable Google Chrome for Android

**Operating system:** Android v6.0.1

### PR DESCRIPTION
**Pull request by Arpit-Gulgulia**
Signed-off-by: Arpit-Gulgulia <arpitgulgulia@gmail.com>

### Description

Please describe your pull request: Fix for 13773 

Fixes #13773 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by Arpit-Gulgulia** [comment]
After changes, the page looks like the attached screenshot on a small screen and a large screen.
**Mobile view:**
image

**Desktop view:**
image

Hope this will helps the users.

@williamdes , please review the changes.
Thanks

**Review by williamdes — APPROVED** [review]
Nice one, it can probably go to QA
Will test it soon

**Comment by Arpit-Gulgulia** [comment]
> Nice one, it can probably go to QA Will test it soon

Hey @williamdes, did you get some time to test this? Hope it passed🙂
Please also add a hacktober-accepted tag.
Thanks!

File: templates/config/form_display/display.twig [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <ul class="nav nav-tabs" id="configFormDisplayTab" role="tablist">
     {% for id, name in tabs %}
       <li class="nav-item" role="presentation">
-        <a class="nav-link{{ loop.first ? ' active' }}" id="{{ id }}-tab" href="#{{ id }}" data-bs-toggle="tab" role="tab" aria-controls="{{ id }}" aria-selected="{{ loop.first ? 'true' : 'false' }}">{{ name }}</a>
+        <a class="nav-link{{ loop.first ? ' active' }} mx-0 px-3 rounded-0" id="{{ id }}-tab" href="#{{ id }}" data-bs-toggle="tab" role="tab" aria-controls="{{ id }}" aria-selected="{{ loop.first ? 'true' : 'false' }}">{{ name }}</a>
```
Comment (MauricioFauth) [inline templates/config/form_display/display.twig:12]: That is not needed.
```suggestion
        <a class="nav-link{{ loop.first ? ' active' }}" id="{{ id }}-tab" href="#{{ id }}" data-bs-toggle="tab" role="tab" aria-controls="{{ id }}" aria-selected="{{ loop.first ? 'true' : 'false' }}">{{ name }}</a>
```

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Comment by Arpit-Gulgulia** [comment]
@MauricioFauth , are we good the changes?

**Review by MauricioFauth — APPROVED** [review]

**Pull request merged**